### PR TITLE
feat: Custom print layout CSS rules

### LINKED ISSUE #18: Add custom `print` layout CSS rules
Add custom print CSS rules to make the web pages print nicer. The top `nav` links should probably be hidden. The SVG images on the front page should also probably be hidden or resized to be smaller?

### PR DESCRIPTION
**Pull request by s-vamshi**
Closes #18 
Custom styles for print added.

Landing Page: [Ronin.pdf](https://github.com/ronin-rb/ronin-rb.github.io/files/12847102/Ronin.pdf)

Install Page: [Install _ Ronin.pdf](https://github.com/ronin-rb/ronin-rb.github.io/files/12847123/Install._.Ronin.pdf)

FAQ Page: [Frequently Asked Questions _ Ronin.pdf](https://github.com/ronin-rb/ronin-rb.github.io/files/12847100/Frequently.Asked.Questions._.Ronin.pdf)

Docs Page: [Documentation _ Ronin.pdf](https://github.com/ronin-rb/ronin-rb.github.io/files/12847106/Documentation._.Ronin.pdf)

Blog Page: [Project Blog _ Ronin.pdf](https://github.com/ronin-rb/ronin-rb.github.io/files/12847104/Project.Blog._.Ronin.pdf)

### CODE REVIEW

**Comment by s-vamshi** [comment]
@postmodern please review print PDFs and let me know if there are any changes to be made.

File: stylesheets/print.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+svg {
+    width: 10%;
+    height: 10%;
+}
+
+/* terminal svg's */
+object.demo {
+    /* display: none; */
```
Comment (postmodern) [inline stylesheets/print.css:37]: Remove commented out code.

File: stylesheets/print.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    display: none;
+}
+
+#ruby-logo {
+    width: 7%;
+    height: 7%;
+    float: right;
+}
+
+#social-media-links svg{
```
Comment (postmodern) [inline stylesheets/print.css:11]: Add space before `{` for consistency.

**Review by postmodern — CHANGES_REQUESTED** [review]
Minor code linting changes needed.

**Comment by postmodern** [comment]
@s-vamshi Looks good, noticed a few CSS code linting issues. Also, could you upload a Print Preview of a Blog Post page?

File: stylesheets/print.css (hunk 1 above)
Comment (s-vamshi) [reply stylesheets/print.css]: Done

File: stylesheets/print.css (hunk 2 above)
Comment (s-vamshi) [reply stylesheets/print.css]: Done and committed the changes

**Comment by postmodern** [comment]
Appears that the `post-links` on each blog post, which contains the SVG icons for Redd/HackerNews/Lobsters/etc, are rendered too big. `article.post .post-links` can be hidden entirely.
[Ronin is participating in Hacktoberfest 2023! _ Ronin.pdf](https://github.com/ronin-rb/ronin-rb.github.io/files/12851803/Ronin.is.participating.in.Hacktoberfest.2023._.Ronin.pdf)

**Review by postmodern — CHANGES_REQUESTED** [review]
Hide `article.post .post-links` to hide the SVG icons for Reddit/HackerNews/Lobsters/etc.

**Comment by s-vamshi** [comment]
@postmodern can you check now?

**Comment by postmodern** [comment]
Looks good!
[Ronin is participating in Hacktoberfest 2023! _ Ronin.pdf](https://github.com/ronin-rb/ronin-rb.github.io/files/12851913/Ronin.is.participating.in.Hacktoberfest.2023._.Ronin.pdf)

**Review by postmodern — APPROVED** [review]

**Comment by postmodern** [comment]
Hacktoberfest 2023 is now over! Congratulations on successfully submitting a PR to Ronin. As mentioned in the [blog post](https://ronin-rb.dev/blog/2023/09/26/ronin-is-participating-in-hacktoberfest-2023.html), you are now eligible for **free** [Ronin stickers](https://merch.ronin-rb.dev/category/stickers)! In order to claim your free stickers, please join the [Ronin Discord server](https://discord.gg/Cw5WWnycRH), and DM me the following information:

1. Which stickers do you want?
2. Your full mailing address.

The stickers will then be mailed in a regular flat-rate envelope, from the US, without tracking. Note, it may take up to one month for International mail to arrive.

**Pull request merged**